core/src/main/java/pl/agh/edu/generator/client_generator/ClientGenerator.java

```diff
@@ -53,7 +59,7 @@ private LocalDateTime getCheckOutTime(int numberOfNight, LocalTime checkOutMaxTi
 	}
 
 	private BigDecimal getDesiredPricePerNight(RoomRank desiredRoomRank, RoomCapacity roomCapacity) {
-		double meanPrice = JSONClientDataLoader.averagePricesPerNight.get(desiredRoomRank).get(roomCapacity).doubleValue();
+		double meanPrice = JSONClientDataLoader.averagePricesPerNight.get(desiredRoomRank).get(roomCapacity).doubleValue() * difficultyMultiplier;
```

Comment:
to jest maksymalna cena jaką klient chce zapłacić więc im trudniejszy poziom tym klient powinien chcieć mniej zapłacić więć dificultyMultiplier jest na odwrót